### PR TITLE
Add bar chart component

### PR DESCRIPTION
**Pull request by graemeRharvey**

### CODE REVIEW

File: server/dummyData.js [hunk 1]
```diff
@@ -1,10 +1,87 @@
 import TestRun from './models/testRun';
 const resultHelper = require('./controllers/helpers/resultHelper');
+const faker = require('faker/locale/en');
+const sample = require('lodash.sample');
+
+// Define dataset restrictions
+const frameworks = ['Cypress', 'Mamba', 'Jest'];
+const results = ['success', 'failure'];
```
Comment (drewbrend) [inline server/dummyData.js:8]: 'skipped' will be a result going forward, though I suppose we can add that here when we get to that stage

File: server/controllers/run.controller.js [hunk 2]
```diff
@@ -71,12 +71,15 @@ function handleResultJson(resultJson, runId) {
...
 export function getRuns(req, res) {
-  TestRun.find(req.body).sort('-runDate').exec((err, runs) => {
-    if (err) {
-      res.status(500).send(err);
-    }
-    res.json({ runs });
-  });
+  TestRun.find(req.body)
+         .sort('-runDate')
+         .populate('results')
```
Comment (drewbrend) [inline server/controllers/run.controller.js:76]: so easy.  that's crazy

File: client/modules/Run/components/RunListItem/RunListItem.js [hunk 3]
```diff
@@ -3,20 +3,24 @@ import PropTypes from 'prop-types';
...
   return (
     <div className={styles['single-test']}>
       <h3 className={styles['test-title']}>
         <Link to={`/runs/${props.run._id}`} >
           {props.run.job}
         </Link>
       </h3>
-      <p className={styles['author-name']}><FormattedMessage id="testType" /> {props.run.results}</p>
-      <p className={styles['author-name']}><FormattedMessage id="testStability" /> {props.run.framework}</p>
+      <p className={styles['author-name']}><FormattedMessage id="testResult" /> {(counts.failure > 0) ? 'FAILED' : 'PASSED'}</p>
```
Comment (drewbrend) [inline client/modules/Run/components/RunListItem/RunListItem.js:22]: should FAILED and PASSED be localized?

Comment (graemeRharvey) [reply client/modules/Run/components/RunListItem/RunListItem.js:22]: yes. lol

Comment (drewbrend) [reply client/modules/Run/components/RunListItem/RunListItem.js:22]: cool beans.

**Comment by drewbrend** [comment]
This repo has been moved into the plangrid domain.  Closing.  https://github.com/plangrid/testboy/pull/3

**Pull request closed without merging**